report/templates/activity.html

```diff
@@ -2,11 +2,70 @@
 
 {% block content %}
 <h2 id="recent_activity"><a href="#recent_activity">Recent activity</a></h2>
-{# TODO: Print message instead of empty graph if no recent activity in repo #}
-<img src="recent_activity.png" alt="Recent activity graph">
+
+<div id="chart_activity" style="border: 1px solid #808080; width: 1014px"><svg style="height: 200px; width: 100%"></svg></div>
+	<script>
+		(async() => {
+			// Fetch the data and wait for it
+			const request = await fetch('recent_activity.dat', {mode: 'no-cors'});
```

Comment:
If I understand correctly, setting `{mode: 'no-cors'}` is a security gap in case someone wants to deploy the generated report via a webserver. Am I right?

Reply:
Well, yes, but the file is local anyway.
For Chrome it looks like it would need the "--allow-file-access-from-files" command line option.

I think I will try a different approach: wrap the json data in a javascript file that sets it in a global variable. This way it can be imported using a <script/> tag. Or maybe we can embed the json data directly in the html and don't use fetch at all.

Reply:
> Or maybe we can embed the json data directly in the html and don't use fetch at all.

something similar, I wanted to suggest: generate a set of js files with data embedded. Suck approach is used by Bokeh and pygal, if I remember correctly

Reply:
`--allow-file-access-from-files` this did not help in chrome, btw and firefox does not display new graphs neither

Reply:
fetch API now removed. Let me know if it works for you.

Reply:
great! It has indeed worked out!